src/Symfony/Component/Security/Core/User/InMemoryUserChecker.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\User;
+
+use Symfony\Component\Security\Core\Exception\AccountExpiredException;
+use Symfony\Component\Security\Core\Exception\CredentialsExpiredException;
+use Symfony\Component\Security\Core\Exception\DisabledException;
+use Symfony\Component\Security\Core\Exception\LockedException;
+
+/**
+ * Checks the state of the in-memory user account.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
```

Comment:
This class is a copy/paste of UserChecker.php, right?

Reply:
right :)